src/tests/JIT/Regression/JitBlue/Runtime_62692/Runtime_62692.cs

```diff
@@ -39,8 +40,8 @@ public static int TestEntryPoint()
             AssertEqual(Problem2(1111, 0xFFFF_FFFF_0000_0001), 3414328792);
             AssertEqual(Problem3(1, 0xFFFF_0001), 0);
             AssertEqual(Problem4(1111, 0xFFFF_FFFF_0000_0001), 3414328792);
-            AssertEqual(Problem5(1111, double.MaxValue), 3307008522);
```

Comment:
Could you explain which change triggered updating the baseline? For mono, the intrinsics was created at https://github.com/dotnet/runtime/blob/main/src/mono/mono/mini/simd-intrinsics.c#L4934-L4939. Then further processed to emit LLVM IR at https://github.com/dotnet/runtime/blob/main/src/mono/mono/mini/mini-llvm.c#L10088-L10103 
We probably needs to emit a different intrinsics to match this new behavior.

Reply:
Hi,
.NET so far had different behavior for floating point to integer conversions on different platform. This effort is to standardize saturating behavior across Intel arch. I am still looking into mono implementation. Feel free to drop comments if there is something which can help correcting the baseline on mono for ```x86/x64```

Reply:
I don't think its quite that simple.

The platform specific hardware intrinsics need to remain as is, while the xplat intrinsics and the general scalar conversions need to be updated to perform saturation.

IIRC, LLVM has a dedicated way to represent saturating conversions as part of its IR. While the Mono JIT would need to ensure the relevant helper call is emitted instead (they do not support AVX today).

I wouldn't expect that to be a lot of work, but I'm not personally familiar with where that work needs to happen.

Reply:
Thanks Tanner.

 Hopefully @fanyang-mono can make the required change. That would be simpler than Kushal looking into mono code.

Reply:
Oh, I see. It is `(uint)data` behaving differently with this PR, not the `Sse42.Crc32`. I will have to look into it to see where the code lives for that type conversion. Feel free to disable the test and open a github issue for it, so that this PR is not block by it.

Reply:
If that's acceptable to the Mono team, it's fine with me.